tests/test_oic_consumer_logout.py

```diff
@@ -271,9 +269,11 @@ def test_logout_without_sub(self):
 
     def test_logout_with_none(self):
         # Now for the backchannel logout. This happens on the OP
-        # Pgitython version 3.5 doesn't support variable annotation
+        # Python version 3.5 doesn't support variable annotation
```

Comment:
You can still use comment annotation as in the rest of the code:
`logout_info = {}  # typing: Dict[str, ...]`

Reply:
Noop! If I do then pylama complains.